app/integration_action.go

```diff
@@ -249,10 +249,17 @@ func (a *App) DoPostActionWithCookie(c *request.Context, postID, actionId, userI
 	defer resp.Body.Close()
 
 	var response model.PostActionIntegrationResponse
-	if err := json.NewDecoder(resp.Body).Decode(&response); err != nil {
+	respBytes, err := ioutil.ReadAll(resp.Body)
```

Comment:
Nit: `ioutil` is deprecated 
```suggestion
	respBytes, err := io.ReadAll(resp.Body)
```

Reply:
Govet was complaining because I used io. Not sure if we must do some update to govet.